.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: guardian/actions-assume-aws-role@v1.0.0
+        with:
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+      - name: Use Node.js 14.15.5
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: 14.15.5
+      - name: Set up JDK 11
```

Comment:
It looks like we are [still using Java 8 in TeamCity](https://teamcity.gutools.co.uk/admin/editBuildParams.html?id=buildType:Tools_Prism)(?), so perhaps we should stick with that for the initial migration:

![image](https://user-images.githubusercontent.com/19384074/135063064-371e2e0a-eece-4674-bcef-cc92ca273657.png)

Aside: it wasn't that easy to figure this out in TeamCity, hooray for moving this config into git!

Reply:
Whoa! Great spot! Updated.

> Aside: it wasn't that easy to figure this out in TeamCity, hooray for moving this config into git!

🎉